plugins/modules/aci_aep_to_epg.py

```diff
@@ -0,0 +1,347 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: aci_aep_to_epg
+short_description: Bind EPG to AEP (infra:RsFuncToEpg).
+description:
+- Bind EPG to AEP.
+options:
+  aep:
+    description:
+    - The name of the Attachable Access Entity Profile.
+    type: str
+    aliases: [ aep_name ]
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  ap:
+    description:
+    - Name of an existing application network profile, that will contain the EPGs.
+    type: str
+    aliases: [ app_profile, app_profile_name ]
+  epg:
+    description:
+    - The name of the end point group.
+    type: str
+    aliases: [ epg_name ]
+  encap_id:
+    description:
+    - The VLAN associated with this application EPG.
+    type: int
+    aliases: [ vlan, vlan_id ]
+  primary_encap_id:
```

Comment:
I would use primary_encap here and have primary_encap_id as an other alias. ( I have seen both in our existing modules we will align them in future)